database/migrations/20190212092504_cards.js

```diff
@@ -1,7 +1,7 @@
 
 exports.up = function(knex, Promise) {
  return knex.schema.createTable('tasks', (table) => { 
-  table.increments('task_id').primary();
+  table.increments().primary();
```

Comment:
Don't edit migration files. Create a new migration file with the changes added into it.